actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadBackGroundJobActor.java

```diff
@@ -1094,6 +1085,15 @@ private void processUserInfo(
               failureUserReq.add(userMap);
               continue;
             }
+            // update user-org table(role update)
+            userMap.put(JsonKey.UPDATED_BY, updatedBy);
+            Util.upsertUserOrgData(userMap);
+            try {
+              // update the user external identity data
+              Util.updateUserExtId(userMap);
+            } catch (Exception ex) {
+              ProjectLogger.log("Exception occurred while updating user external Identity", ex);
```

Comment:
From:
"Exception occurred while updating user external Identity"

To:
"BulkUploadBackGroundJobActor:processUserInfo: Exception occurred in add, update or delete of user external identities."